docs/UserGuide.md

```diff
@@ -341,7 +445,7 @@ You can delete all person entries from the patient book.
 
 <br>
 
-### Appointment Commands
+### **Appointment Commands**
 
 An **appointment** is defined by several fields. The fields and their corresponding prefixes are as follows:
```

Comment:
Minor formatting error, but could you help add a line break after the prefix table (before line 460), and remove the line break after line 460? This is to keep the format consistent with the person commands section.